src/components/Footer/Footer.js

```diff
@@ -38,18 +37,8 @@ const Footer = ({
   blockStatus,
   loadingVote,
   showScoreButton = false,
-  onPressScore,
-  isShowDM = false
+  onPressScore
 }) => {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Several props and states related to direct messages (DMs) have been removed, including `isShowDM`, `sendMessageDM`, `loading`, and `createSignChat`. Make sure these are not required elsewhere in your application, as their removal could lead to unexpected behavior or crashes if they're still being referenced.
```diff
-  onPressScore,
-  isShowDM = false
+  onPressScore
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->